phys/module_mp_nssl_2mom.F

```diff
@@ -1012,45 +1013,48 @@ SUBROUTINE nssl_2mom_init(  &
       
       DO j = 0,nqiacralpha
       alp = float(j)*dqiacralpha
-      y = gamma_sp(1.+alp)
+      y = gamma_dpr(1.+alp)
+      y2 = gamma_dpr(real(2.+alp))
```

Comment:
@MicroTed 
Ted,
Why are these two lines different? Why does one need `real` and the other does not?
```
      y = gamma_dpr(1.+alp)
      y2 = gamma_dpr(real(2.+alp))
```